packages/build/justfile

```diff
@@ -0,0 +1,121 @@
+# Note: this justfile should work for anyone using MacOS, however with some work we can probably get it to also work on
```

Comment:
This is really great! I played around with the commands on this branch and so far so good. I haven't utilized `LABKEY_HOME` as a variable so I had to introduce that.

I'd like to see it merged so I can make use of it during my next development cycle(s).